### PR TITLE
Product overview

### PR DESCRIPTION
**Pull request by robertvgarcia**
Added StyleSelector and AddToCart components

### CODE REVIEW

File: client/src/components/ProductOverview/ProductOverview.jsx [hunk 1]
```diff
@@ -4,24 +4,27 @@ import ImageGallery from './ImageGallery';
...
 import s from './style.css';
 import axios from 'axios';
 
 const ProductOverview = () => {
-  const [styleInfo, setStyleInfo] = useState([]);
+  const [data, setData] = useState([]);
   const [productInfo, setProductInfo] = useState({});
-  const id = 65633;
-  // const id = Math.floor((Math.random() * 1011) + 65631)
+  const id = Math.floor((Math.random() * 1011) + 65631);
```
Comment (keewook2) [inline client/src/components/ProductOverview/ProductOverview.jsx:14]: for id, do we want to get the random item when we first enter webpage? or are we gonna let users type specific item on endpoint? 
if former, we can use this, but if latter, we should use useParam or querystring I think

Comment (keewook2) [reply client/src/components/ProductOverview/ProductOverview.jsx:14]: or if it's temporary one, we can fix it later :)

File: client/src/components/ProductOverview/AddToCart.jsx [hunk 2]
```diff
@@ -5,7 +5,23 @@ const AddToCart = () => {
 
   return (
     <div id={s.cart}>
-      <h1>AddToCart</h1>
+      <select id={s.size}>
+        <option>SELECT SIZE</option>
```
Comment (keewook2) [inline client/src/components/ProductOverview/AddToCart.jsx:9]: Make sure as we talked in the zoom, once you've done with handling style selector stuff, change hardcoding please 👍

**Review by keewook2 — APPROVED** [review]

File: client/src/components/ProductOverview/ProductOverview.jsx (hunk 1 above)
Comment (robertvgarcia) [reply client/src/components/ProductOverview/ProductOverview.jsx:14]: Yes, random product id is temporary, will change it to use Params later

**Pull request merged**